### PR TITLE
New test added + travis dep line

### PR DESCRIPTION
**Pull request by josephramsay**
Reset of https://github.com/linz/postgresql-tableversion/pull/70

### CODE REVIEW

**Comment by strk** [comment]
The test is only failing because the "expected output" was not updated (this would not be needed if we switched to `pgprove` for running tests, but right now we're still using pg_regress).

BTW, the `group: deprecated-2017Q4` line is probably unneeded and it doesn't even fix the warning recommending its use ...

**Comment by strk** [comment]
pg_prove PR is ready, can you rebase ? Sorry about the test file renaming.

**Comment by strk** [comment]
@josephramsay so I did the rebase, it's now clear that the test does not fail. Still worth keeping the test ? If so we should maybe put it into a separate file, like `test/sql/issues/75.pg` ...

**Comment by jrlinz** [comment]
Agree, since the test no longer fails we probably don't need it anymore. I'll let you decide whether or not to keep it or not

**Comment by strk** [comment]
Tests are always good, the only reason not to merge this would be if it is a duplicated test. If it is a new test, I think it should be merged.

File: test/sql/base.pg [hunk 1]
```diff
@@ -454,6 +454,59 @@ SELECT has_function( 'table_version', 'ver_version'::name );
 
 SELECT has_function( 'table_version', 'ver_enable_versioning', ARRAY['regclass'] );
 
+-- Added after problem found in admin_bouldaries_uploader
```
Comment (strk) [inline test/sql/base.pg]: Maybe add a link here to the issue, to have a background of the error condition ?

**Pull request merged**